stdlib/3/urllib/request.pyi

```diff
@@ -222,6 +282,73 @@ class URLopener:
         reporthook: Optional[Callable[[int, int, int], None]] = ...,
         data: Optional[bytes] = ...,
     ) -> Tuple[str, Optional[Message]]: ...
+    def addheader(self, *args: Tuple[str, str]) -> None: ...  # undocumented
+    def cleanup(self) -> None: ...  # undocumented
+    def close(self) -> None: ...  # undocumented
+    def http_error(
```

Comment:
I think I'd simplify these return types to just `_UrlopenRet` aka `Any`.
I don't yet see code paths where we return None and in general typeshed tries to avoid complicated unions in return types (which is probably why `_UrlopenRet` is aliased to `Any` in the first place).

Reply:
Also, I think the `IO[str]`s here should be changed to `IO[bytes]` since they seem to be fed into HTTPError's, as mentioned above